### ISSUE TITLE
User-driven skinning

### ISSUE THREAD

**Issue by Splitwirez** [waliarubal/Jaya#75]
As previously discussed, I think we should allow the user to create and compile their own skins for Jaya File Manager - some people may not be fond of its default look and feel, so by allowing skinning, we enable the user to customize Jaya's appearance pretty much to their heart's content, as long as we do a good job of it.

@waliarubal We agreed that if this was implemented at all, it would be a v1.1 or later feature, correct?

**Comment by waliarubal** [comment]
@Splitwirez  That's correct, we will provide a customization screen in app settings where user will be able to tweak all parts of UI including colors, background, etc.

**Comment by Splitwirez** [comment]
@waliarubal I don't just mean colours and images. I'm suggesting letting the user write XAML, compile it into an assembly, and load that. Like what File Commander was going to do.

**Comment by Splitwirez** [comment]
Recap from discussion over Skype: The idea of implementing colour and background image customization as options for the default skin, rather than for Jaya itself, was discussed. This would allow both 'easy' and 'in-depth' customization to coexist, without forcing all skins to support these extra features.
